products/others/abares_clum_2023/abares_clum_2023.stac-item.json

```
@@ -0,0 +1,437 @@
+          154.13459096827685,
+          -28.926806951171656
+        ],
+        [
+          153.47486260588514,
+          -25.588747716439443
+        ],
+        [
+          152.85298285718966,
+          -22.240505418289334
+        ],
+        [
+          152.26581257920566,
+          -18.871879337594976
+        ],
+        [
+          151.7105476219974,
+          -15.472985813935622
+        ],
+        [
+          151.18467566074924,
+          -12.033926797208792
+        ],
+        [
+          150.68593948225757,
+          -8.544475534300076
+        ],
+        [
+          147.19179609022606,
+          -9.040128864826936
+        ],
+        [
+          143.6746726361613,
+          -9.432813337076608
+        ],
+        [
+          140.13971775361486,
+          -9.722213077781753
+        ],
+        [
+          136.59224130432418,
+          -9.908079230475378
+        ],
+        [
+          133.03766783675908,
+          -9.990244726328434
+        ],
+        [
+          129.48148744269184,
+          -9.968633895468965
+        ],
+        [
+          125.92920510885354,
+          -9.84326674830413
+        ],
+        [
+          122.38628972478618,
+          -9.614257847499111
+        ],
+        [
+          118.8581239227339,
+          -9.281809791722829
+        ],
+        [
+          115.34995588982498,
+          -8.84620143025105
+        ],
+        [
+          114.904191188414,
+          -12.338971974664272
+        ],
+        [
+          114.43405602677457,
+          -15.78248677872958
+        ],
+        [
+          113.93750954399407,
+          -19.187026514039353
+        ],
+        [
+          113.41227816184816,
+          -22.56257992200888
+        ],
+        [
+          112.85582171140769,
+          -25.91916416661671
+        ],
+        [
+          112.26529351896299,
+          -29.267166517190216
+        ],
+        [
+          111.63749316865443,
+          -32.61774170741351
+        ],
+        [
+          110.96881034178841,
+          -35.983311908204406
+        ],
+        [
+          110.2551577253937,
+          -39.37824077722237
+        ],
+        [
+          109.49189045629166,
+          -42.81979963437641
+        ]
+      ]
+    ]
+  },
+  "links":[
+    {
+      "rel":"self",
+      "href":"s3://dea-public-data-dev/abares_clum_2023/abares_clum_2023.stac-item.json",
+      "type":"application/json"
+    },
+    {
+      "rel":"via",
+      "href":"http://www.doi.org/10.25814/2w2p-ph98",
+      "type":"text/html;charset=utf-8"
+    },
+    {
+      "rel":"preview",
+      "href":"abares_clum_2023.thumbnail.jpg",
+      "type":"image/jpeg"
+    }
+  ],
+  "assets":{
+    "thumbnail":{
+      "href":"abares_clum_2023.thumbnail.jpg",
+      "roles":[
+        "thumbnail"
+      ],
+      "type":"image/jpeg"
+    },
+    "alum_class":{
+      "href":"https://dea-public-data-dev.s3-ap-southeast-2.amazonaws.com/abares_clum_2023/clum_50m_2023.tiff",

```

```suggestion
      "href":"clum_50m_2023.tiff",
```
This is better as a relative URL, to simplify downloading or moving the dataset around.

---

Awesome, thank-you!